### DISCUSSION TITLE
Prompt at bottom

### DISCUSSION THREAD

**Discussion by semi710** [starship/starship#5132] [Q&A]
is it possible to pin the prompt at the bottom of the screen 
and all commands executed goes up with the char before that.

**Comment by mickimnet** [comment]
Take a look at transient prompt, maybe that is what you are looking for–see: [Documentation -> Advanced Configuration](https://starship.rs/advanced-config/)

**Reply by semi710** [reply]
> Take a look at transient prompt, maybe that is what you are looking for–see: [Documentation -> Advanced Configuration](https://starship.rs/advanced-config/)

I gave it a look but unable to figure out how to acheive that....
and also was not able to figure out where to put starship.lua

**Reply by mickimnet** [reply]
What Operating System and what shell are you using?

**Reply by semi710** [reply]
Sorry for the incomplete info...

I am using arch Linux with zsh shell

**Reply by mickimnet** [reply]
Ah! Ok, the transient prompt is not yet available with Zsh😕
